test/unit/mdc-dialog/mdc-dialog.test.js

```diff
@@ -73,22 +70,6 @@ function hasClassMatcher(className) {
 
 suite('MDCDialog');
 
-const originalLayout = MDCRipple.prototype.layout;
-const stubbedLayout = td.func('MDCRipple#layout');
-
-before(() => {
-  MDCRipple.prototype.layout = stubbedLayout;
```

Comment:
Remove import of MDCRipple 
`  24:9  error  'MDCRipple' is defined but never used. Allowed unused vars must match /^(?:(?:MDC(?:(?:[A-Z][a-z0-9]+)+)Adapter)|(?:(?:(?:[A-Z][a-z0-9]+)+)Type))$/  no-unused-vars`